lib/mercadopago/api/cards.rb

```diff
@@ -3,31 +3,33 @@
 require '../base'
 
 module MercadoPago
-  class Cards < Base
-    # Override parent class method
-    # Place here main base URL
-    def service_url
-      'https://api.mercadopago.com'
-    end
+  module API
+    class Cards < Base
+      # Override parent class method
+      # Place here main base URL
+      def service_url
+        'https://api.mercadopago.com'
+      end
 
-    # GET /v1/customers/:customer_id/cards
-    # It retrieves all cards from a customer
-    def retrieve_customer_cards(customer_id)
-      response = connection.get customer_cards_endpoint(customer_id) do |req|
-        req.params['access_token'] = @access_token
+      # GET /v1/customers/:customer_id/cards
+      # It retrieves all cards from a customer
+      def retrieve_customer_cards(customer_id)
+        response = connection.get customer_cards_endpoint(customer_id) do |req|
+          req.params['access_token'] = @access_token
+        end
+        response = process_response(response)
+        OpenStruct.new(success?: true, body: response)
+      rescue Faraday::ClientError => exception
+        OpenStruct.new(success?: false, message: 'Los datos no son correctos', details: exception.response[:body].to_s)
+      rescue Faraday::Error::TimeoutError, Faraday::ConnectionFailed, Timeout::Error => e
```

Comment:
MercadoPago::API::Cards#retrieve_customer_cards has the variable name 'e'

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)